Dangerfile

```diff
@@ -62,8 +62,12 @@ junit.report
 all_test = junit.tests.map(&:attributes)
 slowest_test = all_test.sort_by { |attributes| attributes[:time].to_f }.last
 
-message = "### Slowest tests \n\n"
-message << "File | Name | Time |\n"
-message << "| --- | ----- | ----- |\n"
-message << "| #{slowest_test[:file]} | #{slowest_test[:name]} | #{slowest_test[:time]} |\n"
-markdown message
+if !slowest_test.nil?
```

Comment:
I'm not sure that we need this check here. If there is no unit test report it means that something went wrong.
For testing coverage I would suggest to not drop the testExample that we had, but don't call mainfunc from there instead. So there will be a test but it won't make any hits to framework codebase..